docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -13,17 +13,23 @@ Provides blocks related actions.
 === "Syntax"
 
     ```bash
-    besu blocks import --from=<block-file>
+    besu blocks import [--skip-pow-validation-enabled] --from=<block-file>
     ```
 
 === "Example"
 
     ```bash
-    besu blocks import --from=/home/me/me_project/mainnet.blocks
+    besu blocks import --skip-pow-validation-enabled --from=/home/me/me_project/mainnet.blocks
     ```
 
 Imports blocks from the specified file into the blockchain database.
 
+Including `--skip-pow-validation-enabled` enables skipping the validation of the `mixHash` when importing blocks.
```

Comment:
Minor suggestion: "Including `--skip-pow-validation-enabled` skips validation of the `mixHash` when importing blocks.